### ISSUE TITLE
 Accept several orders in one click 

### ISSUE THREAD

**Issue by rayon9coop** [coopcycle/coopcycle-web#926]
When a store owner encodes several delivery requests, validation one by one is laborious. It might be useful to link these requests to validate them in one click.

**Comment by rayon9coop** [comment]
Another way to do: automaticaly accept and confirm a task if it’s assigned

**Comment by Paul-Eraman-CoopCycle** [comment]
Do you mean accept deliveries as an administrator or close a task as a rider?

**Comment by rayon9coop** [comment]
I mean accept orders as an administrator. 
Maybe this is more accurate : automaticaly accept and confirm when the task is assigned to a rider.

**Comment by Paul-Eraman-CoopCycle** [comment]
These buttons should be clickable and acept the order. 

if you hover over the "new" button it should turn a light transparent green and the text should change to "accept". if clicked, it turns darker green and is no longer a button

<img width="1582" alt="Screenshot 2024-03-28 at 16 18 04" src="https://github.com/coopcycle/coopcycle-web/assets/77277854/ea979f69-a3c4-416e-834a-9fc538f87787">

**Comment by rayon9coop** [comment]
> These buttons should be clickable and acept the order.
> 
> if you hover over the "new" button it should turn a light transparent green and the text should change to "accept". if clicked, it turns darker green and is no longer a button
> 
> <img alt="Screenshot 2024-03-28 at 16 18 04" width="1582" src="https://private-user-images.githubusercontent.com/77277854/317783941-ee3488d3-539f-421e-87a9-ea7c098a5be8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE2NDY3MjAsIm5iZiI6MTcxMTY0NjQyMCwicGF0aCI6Ii83NzI3Nzg1NC8zMTc3ODM5NDEtZWUzNDg4ZDMtNTM5Zi00MjFlLTg3YTktZWE3YzA5OGE1YmU4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI4VDE3MjAyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZjZTYwMTk0NTYxMDM5ODFjOWNkOGM1MTg4NTc5ODBmZGQ2ZTEzNDIzY2IyMWI5NzI1NDY3YzUxYjNkZGI3NTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-O8Ta18DD738ZvoMPRrGgUKnNs-blTWrAMG0vN-IStA">

Yes, that's a good idea. I still think it's a good idea for the order to be automatically confirmed as soon as a dispatcher has assigned the order's tasks to a courier.

**Comment by Paul-Eraman-CoopCycle** [comment]
Yeah also true

**Comment by Paul-Eraman-CoopCycle** [comment]
Design for this is that clicking on the "new" button would accept it
